starter-code/css/main.css

```diff
@@ -4,7 +4,126 @@ dark blue: #06365f
 light blue: #1c5380
```

Comment:
Good job, please see the comments to try to fix the float left and float right issue you have.